src/components/colorbar/attributes.js

```diff
@@ -136,6 +136,7 @@ module.exports = overrideAll({
     tickvals: axesAttrs.tickvals,
     ticktext: axesAttrs.ticktext,
     ticks: extendFlat({}, axesAttrs.ticks, {dflt: ''}),
+    ticklabeloverflow: extendFlat({}, axesAttrs.ticklabeloverflow, {dflt: 'allow'}),
```

Comment:
You don't want a `dflt` here, do you?

Reply:
Good eye. Addressed in https://github.com/plotly/plotly.js/pull/5584/commits/f833ce41f9caafc7faab1ebf00861d573caa5687.